src/main.js

```diff
@@ -1,6 +1,8 @@
 import cli from 'cli';
 import Validator from 'validator';
 
+import 'babel-core/polyfill';
```

Comment:
(I just would have thought webpack handles this for us or it would go in a config.)